### PR TITLE
feat: make home slide fragment recycler view

### PR DESCRIPTION
**Pull request by Gijuno**

### CODE REVIEW

File: app/src/main/java/com/implude/niche/presentation/home/HomeFragment.kt [hunk 1]
```diff
@@ -28,54 +28,37 @@ class HomeFragment : BaseFragment<FragmentHomeBinding>(R.layout.fragment_home) {
...
+        val recommendPlaceLayoutManager = LinearLayoutManager(this.requireContext())
+        recommendPlaceLayoutManager.orientation = LinearLayoutManager.HORIZONTAL
+
+        val nearPopularPlaceAdapter = NearPopularPlaceAdapter(this.requireContext())
+        val nearPopularPlaceLayoutManager = LinearLayoutManager(this.requireContext())
+        nearPopularPlaceLayoutManager.orientation = LinearLayoutManager.HORIZONTAL
+
+        val nearPopularTopicPlaceAdapter = NearPopularTopicPlaceAdapter(this.requireContext())
+        val nearPopularTopicPlaceLayoutManager = LinearLayoutManager(this.requireContext())
+        nearPopularTopicPlaceLayoutManager.orientation = LinearLayoutManager.HORIZONTAL
```
Comment (nutyworks) [inline app/src/main/java/com/implude/niche/presentation/home/HomeFragment.kt:41]: Is there any reason to separate adapter classes?

File: app/src/main/java/com/implude/niche/presentation/home/HomeFragment.kt [hunk 2]
```diff
@@ -24,16 +28,37 @@ class HomeFragment : BaseFragment<FragmentHomeBinding>(R.layout.fragment_home) {
...
+        }
+
+        binding.fragmentHomeSlide.nearPopularTopicRecyclerview.apply {
+            this.layoutManager = nearPopularTopicPlaceLayoutManager
+            this.adapter = nearPopularTopicPlaceAdapter
+
         }
 
-        placeAdapter.notifyDataSetChanged()
+        recommendPlaceAdapter.notifyDataSetChanged()
```
Comment (nutyworks) [inline app/src/main/java/com/implude/niche/presentation/home/HomeFragment.kt:61]: No nofityDataSetChanged method is needed. It is automatically called when data changes. (defined in `BaseRecyclerViewAdapter`)

File: app/src/main/java/com/implude/niche/presentation/home/HomeFragment.kt [hunk 3]
```diff
@@ -24,16 +28,37 @@ class HomeFragment : BaseFragment<FragmentHomeBinding>(R.layout.fragment_home) {
...
+            this.layoutManager = nearPopularPlaceLayoutManager
+            this.adapter = nearPopularPlaceAdapter
+
+        }
+
+        binding.fragmentHomeSlide.nearPopularTopicRecyclerview.apply {
+            this.layoutManager = nearPopularTopicPlaceLayoutManager
+            this.adapter = nearPopularTopicPlaceAdapter
+
         }
```
Comment (nutyworks) [inline app/src/main/java/com/implude/niche/presentation/home/HomeFragment.kt:59]: There is no reason to separate adapter classes with each use.

**Review by nutyworks — CHANGES_REQUESTED** [review]
많이 봐줬다

File: app/src/main/java/com/implude/niche/presentation/home/HomeFragment.kt (hunk 3 above)
Comment (nutyworks) [reply app/src/main/java/com/implude/niche/presentation/home/HomeFragment.kt:59]: From line 31 to line 59.

File: app/src/main/java/com/implude/niche/presentation/home/HomeFragment.kt (hunk 2 above)
Comment (Gijuno) [reply app/src/main/java/com/implude/niche/presentation/home/HomeFragment.kt:61]: wowowowow thx

File: app/src/main/java/com/implude/niche/presentation/home/HomeFragment.kt (hunk 3 above)
Comment (Gijuno) [reply app/src/main/java/com/implude/niche/presentation/home/HomeFragment.kt:59]: It doesn't work with one adapter.. Is there any more solution?

Comment (nutyworks) [reply app/src/main/java/com/implude/niche/presentation/home/HomeFragment.kt:59]: Making multiple adapter instances with one adapter class will work.

Comment (Gijuno) [reply app/src/main/java/com/implude/niche/presentation/home/HomeFragment.kt:59]: Can you refactor the adapter..? I think it could be faster than me

**Review by nutyworks — APPROVED** [review]

**Pull request merged**